src/backend/distributed/sql/citus--10.2-4--11.0-1.sql

```diff
@@ -13,6 +14,7 @@ DROP FUNCTION pg_catalog.master_append_table_to_shard(bigint, text, text, intege
 -- all existing citus local tables are auto converted
 -- none of the other tables can have auto-converted as true
 ALTER TABLE pg_catalog.pg_dist_partition ADD COLUMN autoconverted boolean DEFAULT false;
+ALTER TABLE citus.pg_dist_object ADD COLUMN force_pushdown bool DEFAULT FALSE;
```

Comment:
would it make sense for the default to be NULL? since it's not relevant to many of the objects in pg_dist_object

Reply:
Done.